galasa-parent/dev.galasa.framework.api.resources/src/main/java/dev/galasa/framework/api/resources/routes/ResourcesRoute.java

```diff
@@ -50,34 +48,33 @@ public class ResourcesRoute  extends BaseRoute{
     
     protected List<String> errors = new ArrayList<String>();
 
-    private IFramework framework;
+    private CPSFacade cps;
 
-    public ResourcesRoute(ResponseBuilder responseBuilder,  IFramework framework ) {
+    public ResourcesRoute(ResponseBuilder responseBuilder, IFramework framework) throws ConfigurationPropertyStoreException {
```

Comment:
Pass the cps in, then this class doesn't need the framework ?

Reply:
Good spot, thanks! Done in https://github.com/galasa-dev/framework/pull/638/commits/e0ebeed9d20a210f1213aa84a9603a7cd396b364